app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,33 @@
+class FoodsController < ApplicationController
+  def index
+    @foods = Food.all
+  end
```

Comment:
Here is the only thing that needs modification
In the project requirements, it says that it should display a list of food added by the logged-in user only
So if I log in as a different user I should not be able to see the list.

It can be done by getting the foods that are only created by the current user
@foods = Food.where(user: current_user)

Reply:
Thank you @Peter1907 , I will do the correction.